test_runner/fixtures/zenith_fixtures.py

```diff
@@ -189,9 +193,19 @@ def config(self, lines):
                 conf.write('\n')
 
     def stop(self):
+        """ stop the server """
         if self.running:
             self.zenith_cli.run(['pg', 'stop', self.branch])
 
+    def destroy(self):
```

Comment:
JFYI: Few days ago I've added '--destroy' flag to CLI which does exactly this (with intention to enable that as default behavior later)

Reply:
Ah, cool. I modified the test to use that. Nice to get some test coverage for the '--destroy' flag, too.